addons/mail/static/src/components/message/message.xml

```diff
@@ -114,75 +101,54 @@
                                 </t>
                             </t>
-                            <div class="o_Message_commands o_Message_headerCommands" t-att-class="{ 'o-mobile': messaging.device.isMobile }">
-                                <t t-if="!message.isTemporary and ((message.message_type !== 'notification' and message.originThread and message.originThread.model === 'mail.channel') or !message.isTransient)">
-                                    <span class="o_Message_command o_Message_commandStar o_Message_headerCommand fa"
-                                        t-att-class="{
-                                            'fa-star': message.isStarred,
-                                            'fa-star-o': !message.isStarred,
-                                            'o-message-selected': isSelected,
-                                            'o-message-starred': message.isStarred,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickStar" title="Mark as Todo"
-                                    />
-                                </t>
-                                <t t-if="props.hasReplyIcon">
-                                    <span class="o_Message_command o_Message_commandReply o_Message_headerCommand fa fa-reply"
-                                        t-att-class="{
-                                            'o-message-selected': isSelected,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickReply" title="Reply"
-                                    />
-                                </t>
-                                <t t-if="props.hasMarkAsReadIcon">
-                                    <span class="o_Message_command o_Message_commandMarkAsRead o_Message_headerCommand fa fa-check"
-                                        t-att-class="{
-                                            'o-message-selected': isSelected,
-                                            'o-mobile': messaging.device.isMobile,
-                                        }" t-on-click="_onClickMarkAsRead" title="Mark as Read"
-                                    />
-                                </t>
-                            </div>
                         </div>
                     </t>
-                    <div class="o_Message_content" t-ref="content">
-                        <div class="o_Message_prettyBody" t-ref="prettyBody"/><!-- message.prettyBody is inserted here from _update() -->
-                        <t t-if="message.subtype_description and !message.isBodyEqualSubtypeDescription">
-                            <p t-esc="message.subtype_description"/>
+                    <div class="o_Message_contentWrapper">
+                        <MessageActionList
+                            message="message"
```

Comment:
We had problem in the past giving the reference and not the `localId`, the issue could be somewhat resolved or mitigated now? But I'm not confident in starting to use them. Maybe we can make a pass later.